4.analyze_data/notebooks/UMAP/scripts/1.vis_UMAP.r

```diff
@@ -0,0 +1,142 @@
+suppressPackageStartupMessages(library(ggplot2))
+suppressPackageStartupMessages(library(dplyr))
+
+
+# Set directory and file structure
+umap_dir <- file.path("results")
+umap_files <- list.files(umap_dir, full.names = TRUE)
+print(umap_files)
+
+output_fig_dir <- file.path("figures")
+umap_prefix <- "UMAP_"
+plate_suffix <- "_sc_feature_selected.tsv"
+
+# Define output figure paths as a dictionary where each plate has a figure output path
+output_umap_files <- list()
+for (umap_file in umap_files) {
```

Comment:
This is very pythonic haha Rython

Reply:
Yeah! I noticed it from both Greg and Erik so I want to try and keep this method haha